static/app/views/insights/pages/frontend/settings.ts

```diff
@@ -2,3 +2,10 @@ import {t} from 'sentry/locale';
 
 export const FRONTEND_LANDING_SUB_PATH = 'frontend';
 export const FRONTEND_LANDING_TITLE = t('Frontend');
+
+export const OVERVIEW_PAGE_ALLOWED_OPS = [
+  'pageload',
+  'navigation',
+  'ui.render',
```

Comment:
Is `ui.render` a generic transaction op defined by the SDK, or one of our custom transaction ops? It seems to be defined by a custom transaction related to our smart search bar in our frontend code.

On that topic, this makes me realize that if we don't specify custom transaction ops in here, they will get filtered out. Is there a way to work around this?